src/components/Login/components/AuthErrorsContainer/AuthErrorsContainer.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import styles from './AuthErrorsContainer.css';
+
+export default class AuthErrorsContainer extends Component {
+  static propTypes = {
+    errors: PropTypes.arrayOf(PropTypes.object).isRequired,
+  };
+
+  getErrorMessage(error = {}) {
+    const {
+      code = 'default.error',
+      parameters = [],
+    } = error;
+
+    const values = parameters.reduce((res, { key, value }) => ({ ...res, [key]: value }), {});
+
+    return (
+      <FormattedMessage
+        id={`stripes-core.loginServices.login.${code}`}
+        values={values}
+      />
+    );
+  }
+
+  getPasswordValidationMessages(errors) {
+    return errors.map(element => (
+      <li key={`${element.code}-${element.type}`}>
+        {this.getErrorMessage(element)}
+      </li>
+    ));
+  }
+
+  parseErrors() {
+    const { errors } = this.props;
+
+    return errors.length > 1 ? (
+      <ul className={styles.AuthErrorsList}>{this.getPasswordValidationMessages(errors)}</ul>
+    ) : (
+      this.getErrorMessage(errors[0])
```

Comment:
Are these styles different for list of errors and single error? Why we can't use <ul> for both cases to avoid condition?